webrender/src/internal_types.rs

```diff
@@ -313,14 +315,23 @@ pub struct PackedTexel {
 }
 
 impl PackedTexel {
-    pub fn from_color(color: &ColorF) -> PackedTexel {
+    pub fn high_bytes(color: &ColorF) -> PackedTexel {
         PackedTexel {
             b: (0.5 + color.b * COLOR_FLOAT_TO_FIXED).floor() as u8,
             g: (0.5 + color.g * COLOR_FLOAT_TO_FIXED).floor() as u8,
             r: (0.5 + color.r * COLOR_FLOAT_TO_FIXED).floor() as u8,
             a: (0.5 + color.a * COLOR_FLOAT_TO_FIXED).floor() as u8,
         }
     }
+
+    pub fn low_bytes(color: &ColorF) -> PackedTexel {
+        PackedTexel {
+            b: ((0.5 + color.b * COLOR_FLOAT_TO_FIXED_WIDE).floor() as u16 & 0xff) as u8,
```

Comment:
you could technically have a helper method here, which would accept `COLOR_FLOAT_TO_FIXED_WIDE` or `COLOR_FLOAT_TO_FIXED` as an extra argument